templates/docs/examples/patterns/rule/muted.html

```diff
@@ -8,20 +8,20 @@
 <div class="p-section">
   <div class="row--25-75">
     <div class="col">
-      <hr class="p-rule">
+      <hr>
       <div class="row">
         <div class="col-6">
           <h2>Infrastructure, <br>your way</h2>
           <p>Flexible and engineered for efficiency. <br>From small, private clouds on rails to clouds at any scale.</p>
         </div>
         <div class="col-3">
-          <h2><a href="#">OpenStack&nbsp;›</a></h2>
-          <hr class="p-rule--muted u-hide--medium">
-          <h2><a href="#">Ceph&nbsp;›</a></h2>
-          <hr class="p-rule--muted u-hide--medium">
-          <h2><a href="#">LXD&nbsp;›</a></h2>
-          <hr class="p-rule--muted u-hide--medium">
-          <h2><a href="#">MAAS&nbsp;›</a></h2>
+          <h2><a href="#">OpenStack&nbsp;&rsaquo;</a></h2>
+          <hr class="is-muted u-hide--medium">
```

Comment:
Here and in following lines:
```suggestion
          <hr class="p-rule--muted u-hide--medium">
```

Reply:
Should this also be `p-rule--muted` as Bartek suggested?